ruby/alias_manager.rb

```diff
@@ -17,25 +24,24 @@
 -print the names_and_aliases hash contents in list.      
 =end 
 
-def alias_maker (full_name)
-  
-  full = full_name.split(" ")
-  original_first_name = full[0]
-  original_last_name = full[1]
-  new_name = []
-
-  full.each_index do |x|
-    y = full[x].gsub(/[a-z]/, 
-        "a"=>"e", 'b'=>"c", "c"=>"d", "d"=>"f", "e"=>"i", "f"=>"g", "g"=>"h", "h"=>"j", 'i'=>'o', 'j'=>'k', 
-        'k'=>'l', 'l'=>'m', 'm'=>'n', 'n'=>'p', 'o' =>'u', 'p'=>'q', 'q'=>'r', 'r'=>'s', 's'=>'t', 't'=>'v', 
-        'u' =>'a', 'v'=>'w', 'w'=>'x', 'x'=>'y', 'y'=>'z', 'z'=>'b').capitalize 
-    new_name.unshift(y) 
-  end 
-  new_name.join(" ")
+def alias_maker(full_name)
+  vowels= ["a", "e", "i", "o", "u", "a"]
```

Comment:
Play with ranges on line 28 and 29 to see if you can make these without typing the whole alphabet.

ruby recognizes `("a".."z")` as a valid range